README.md

```diff
@@ -105,7 +105,14 @@ BASE_DOMAIN=your.valid.domain.com
 CLUSTER_NAME=clustername
 # Set your valid DNS VIP, such as 1.1.1.1 for 'ns1.example.com'
 DNS_VIP="1.1.1.1"
+# Set your IP Network stack family, determines the IP address family of the cluster
+# subnet and service subnet, defaults to `v6`
+IP_STACK="v4"
+# Set your host IP Network stack family, determines the IP address family of the provisioning
+# network and external subnets, defaults to the value of `${IP_STACK}`
+HOST_IP_STACK="v4"
 # Set your default network type, `OpenShiftSDN` or `OVNKubernetes`, defaults to `OpenShiftSDN`
+# for IPv4, defaults to `OVNKubernetes` for IPv6 and DualStack
```

Comment:
It might be worth mentioning that ipv6/dual stack will only work with OVNKubernetes?

Reply:
Thanks, done, I hope it's better now.